src/h5cpp/dataspace/points.cpp

```diff
@@ -26,35 +26,72 @@
 //
 #include <h5cpp/dataspace/points.hpp>
 #include <h5cpp/error/error.hpp>
+#include <sstream>
 
-namespace hdf5 {
-namespace dataspace {
+namespace hdf5
+{
+namespace dataspace
+{
 
 Points::Points() :
     Selection() {}
 
 Points::~Points() {}
```

Comment:
Use `=default` and I suggest putting trivial constructors + destructors in the header file.